app/domain/operations/hbx_enrollments/drop_enrollment_members.rb

```diff
@@ -29,17 +30,19 @@ def drop_enrollment_members(params)
         terminate_date = Date.strptime(params[:options]["termination_date_#{params[:hbx_enrollment].id}"], "%m/%d/%Y")
         dropped_enr_members = params[:options].select{|string| string.include?("terminate_member")}.values
         all_enr_members = params[:hbx_enrollment].hbx_enrollment_members
-        eligible_members = all_enr_members.reject{ |member| dropped_enr_members.include?(member.id.to_s) }
+        non_eligible_members = all_enr_members.select{ |member| dropped_enr_members.include?(member.id.to_s) }
+        new_effective_date = terminate_date + 1.day
```

Comment:
This can't be always true,  some times we will needed to drop dependent with base enrollment effective date and new enrollment effective date with the base enrollment effective date

Reply:
This was the specific requirement provided for this project.

Reply:
Please select effective date based on termination date,

termination date <= base_enrollment.effective_on then dep_drop_enrollment effective date = base_enrollment.effective date,

otherwise if termination date > base_enrollment.effective_on then dependent drop enrollment effective date = termination date + 1.day

Reply:
Using 
terminate_date = Date.strptime(params[:options]["termination_date_#{params[:hbx_enrollment].id}"], "%m/%d/%Y")
        termination_date = (terminate_date > params[:hbx_enrollment].effective_on) ? terminate_date + 1.day : terminate_date

To determine termination_date which gives the new effective date of the enrollment.